opentelemetry-sdk/src/metrics/meter_provider.rs

```diff
@@ -310,7 +346,8 @@ mod tests {
             .with_resource(Resource::empty())
             .build();
 
-        assert_service_name(no_service_name, None);
+        assert_resource(&no_service_name, SERVICE_NAME, None);
```

Comment:
nit - as earlier, this test is redundant, and can be removed. As we are checking the length in the next test.